vllm/envs.py

```diff
@@ -296,6 +297,10 @@ def maybe_convert_int(value: Optional[str]) -> Optional[int]:
     "VLLM_LOGGING_LEVEL":
     lambda: os.getenv("VLLM_LOGGING_LEVEL", "INFO"),
 
+    # this is used to configuring logs to redact
+    "VLLM_LOG_FILTER_PATTERNS":
```

Comment:
specifying multiple patterns through env var, is quite complicated, and need to deal with complicated escaping, especially you need to split it by commas.